src/CsvOptions.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Keboola\Csv;
+
+class CsvOptions
+{
+    const DEFAULT_DELIMITER = ',';
+    const DEFAULT_ENCLOSURE = '"';
+    const DEFAULT_ESCAPED_BY = "";
+
+    /**
+     * @var string
+     */
+    private $delimiter;
+
+    /**
+     * @var string
+     */
+    private $enclosure;
+
+    /**
+     * @var string
+     */
+    private $escapedBy;
+
+    /**
+     * @param  string $delimiter
+     * @param  string $enclosure
+     * @param  string $escapedBy
+     * @throws InvalidArgumentException
+     */
+    public function __construct(
+        $delimiter = self::DEFAULT_DELIMITER,
+        $enclosure = self::DEFAULT_ENCLOSURE,
+        $escapedBy = self::DEFAULT_ESCAPED_BY
+    ) {
+        $this->escapedBy = $escapedBy;
+        $this->setDelimiter($delimiter);
+        $this->setEnclosure($enclosure);
+    }
+
+    /**
+     * @param string $delimiter
+     * @throws InvalidArgumentException
+     */
+    protected function setDelimiter($delimiter)
```

Comment:
mozna to muze byt rovnou private?

Reply:
Pak to nemá moc smysl a dal sem všechno do konstruktoru.